src/vma/sock/sockinfo.cpp

```diff
@@ -1293,8 +1293,10 @@ void sockinfo::move_descs(ring* p_ring, descq_t *toq, descq_t *fromq, bool own)
 	mem_buf_desc_t *temp;
 	const size_t size = fromq->size();
 	for (size_t i = 0 ; i < size; i++) {
+		/* coverity[returned_null] */
 		temp = fromq->front();
 		fromq->pop_front();
+		/* coverity[nonnull] */
```

Comment:
[question] Here I see pattern different from other places. I would expect nonnull at assignment and dereference suppression next. Am I wrong?